src/C_TO_LOGIC.py

```diff
@@ -281,14 +281,7 @@ def casthelp(arg):
     pass; 
     
 def LIST_UNION(self_a,b):
-  # Ummm?
-  if self_a == b:
```

Comment:
@QuixoticValentine 
Sometimes these lists can be huge and IIUC doing a list add `self_a + b` will copy contents from both lists to form a new list? 

This func was kind dumb and the first arg named `self` to try and indicate 'its OK to modify the `a` list in place...

Maybe if I had some tests we could really say whats faster :grimacing: 

For now I might say leave this func alone :-/ ?

What do you think?



I think I prefer to append to the existing 

But this old check for dumb 'are you doing a union with itself?' was added for some reason to speed some things up I think and makes me feel suspicious to remove it...

Do you mind leaving that compare and return just in case?

The nice one